### DISCUSSION TITLE
Support deep compression for embedding network with constant width

### DISCUSSION THREAD

**Discussion by youngjae142** [deepmodeling/deepmd-kit#1422] [Ideas]
Plz, support the deep compression feature with potentials with embedding network neurons of constant width.
Currently, multiple hidden-layered potentials with n+1=n*2 size only are supported.
In those cases, layers with layers more than 6 require too much memory, so fail to run.

If the hidden-layer size with n+1=n can be compressed, which utilizes skip connection too, it would be better to make deeper potentials that show better accuracy in some cases.

**Comment by youngjae142** [comment]
Related previous question about the supported hidden-layer size of deep compression
https://github.com/deepmodeling/deepmd-kit/issues/1370#issue-1086388934

**Reply by njzjz** [reply]
ok, I'll mark it as a feature request.
